Index: build.sbt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- build.sbt	(date 1539572172000)
+++ build.sbt	(date 1544007466000)
@@ -10,7 +10,7 @@
 lazy val commonSettings = Seq(
   organization := "edu.berkeley.cs",
   version      := "1.2",
-  scalaVersion := "2.12.4",
+  scalaVersion := "2.11.12",
   crossScalaVersions := Seq("2.12.4", "2.11.12"),
   parallelExecution in Global := false,
   traceLevel   := 15,
Index: src/main/scala/devices/tilelink/DevNull.scala
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/scala/devices/tilelink/DevNull.scala	(date 1539572172000)
+++ src/main/scala/devices/tilelink/DevNull.scala	(date 1544008173000)
@@ -13,7 +13,7 @@
   region: RegionType.T = RegionType.UNCACHEABLE,
   executable: Boolean = true,
   mayDenyGet: Boolean = true,
-  mayDenyPut: Boolean = true,
+  mayDenyPut: Boolean = true
 ) {
   require (1 <= maxAtomic, s"Atomic transfer size must be > 1 (was $maxAtomic)")
   require (maxAtomic <= maxTransfer, s"Atomic transfer size must be <= max transfer (but $maxAtomic > $maxTransfer)")
